packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -38,6 +38,14 @@ function glConstantToRegl(value: ?number): ?string {
   throw new Error(`unhandled constant value ${JSON.stringify(value)}`);
 }
 
+const getSceneToDraw = ({ json }) => {
```

Comment:
should this throw an error if we can't find scene or scenes?

Reply:
I'll make it do that. Maybe it should just not render anything if there are no scenes, but we can relax it later -- throwing is still "more permissive" than current behaviour.